lighthouse-cli/test/smokehouse/test-definitions/screenshot/expectations.js

```diff
@@ -12,9 +12,39 @@ const expectations = [
   {
     artifacts: {
       FullPageScreenshot: {
-        width: '>1000',
-        height: '>1000',
-        data: /data:image\/jpeg;base64,.{10000,}$/,
+        screenshot: {
+          width: '>1000',
+          height: '>1000',
+          data: /data:image\/jpeg;base64,.{10000,}$/,
+        },
+        nodes: {
+          'page-0-BODY': {
+            top: 8,
+            bottom: 1008,
+            left: 8,
+            right: 1008,
+            width: 1000,
+            height: 1000,
+          },
+          'page-1-P': {
+            top: 8,
+            bottom: 26,
+            left: 8,
+            right: 1008,
+            width: 1000,
+            height: 18,
+          },
+          '5-2-BODY': {
```

Comment:
how are these sharing the same incremental map counter when they're in different execution contexts...

shouldn't it start over at `5-0-BODY`?

Reply:
Because different gathers, in the same pass and execution context, are collecting node details on the same elements.

Reply:
oh, and the reason you wouldn't see `5-1` of `5-1` but see `5-2` is perhaps because the first few nodes had zero size.